lib/rucio/tests/test_account.py

```diff
@@ -91,8 +92,8 @@ def test_add_account_attribute(self):
 class TestAccountRestApi():
     def setup(self):
         if config_get_bool('common', 'multi_vo', raise_exception=False, default=False):
-            self.vo_header = {'X-Rucio-VO': 'tst'}
-            self.vo = {'vo': 'tst'}
+            self.vo_header = {'X-Rucio-VO': config_get('client', 'vo', raise_exception=False, default='tst')}
+            self.vo = {'vo': config_get('client', 'vo', raise_exception=False, default='tst')}
```

Comment:
A small thing: For tidiness, I think we should avoid making multiple calls to config_get in the test setup. Instead use the VO to set the VO header. This is in quite a few files: 
- test_account.py
- test_authentication.py
- test_import_export.py
- test_multi_vo.py
- test_replica.py
- test_root_proxy.py
- test_rse.py
- test_rse_expression_parser.py
- test_subscription.py

Reply:
Yeah that totally makes sense, I just did this with a blanket find/replace as I didn't want to change each test manually but it should definitely change as you say.